README.md

```diff
@@ -93,190 +84,10 @@ Here you have a more complete example showcasing all functionality. It includes:
 - A shortcut to roll a dice and get a random number
 - An event handler that echoes reactions to messages.
 - A greeting whenever someone says `hello` in a channel where the bot is present.
+- Home tab example with actions
 >Remember to `export BOT_TOKEN=xoxb-your-bot-secret` to enable slack api calls.
 
 
```

Comment:
I would leave the code here too, many mobile users will just scroll the readme without following any link, so it's nice to showcase all features on the main page

Reply:
Good insight. I'll restore it on the next push